### PR TITLE
Revised the multiplication operator article

### PR DESCRIPTION
**Pull request by pkulikov**
Supported by dotnet/samples#671

@BillWagner I hope the revision of the individual operator articles is inline with the [reference consolidation project](https://github.com/dotnet/docs/projects/71). After going through the articles about individual operators, I would like to review the existing *overview* pages about operators. If you're fine with that, I'll open the dedicated issue to discuss the possible changes later.

### CODE REVIEW

File: docs/csharp/programming-guide/unsafe-code-pointers/how-to-obtain-the-value-of-a-pointer-variable.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "How to: Obtain the Value of a Pointer Variable - C# Programming Guide"
+title: "How to: obtain the value of a pointer variable - C# Programming Guide"
```
Comment (pkulikov) [inline docs/csharp/programming-guide/unsafe-code-pointers/how-to-obtain-the-value-of-a-pointer-variable.md]: Hide white spaces to see that almost nothing has been changed in this article.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

**Review by BillWagner — APPROVED** [review]
Thanks for updating this as well  @pkulikov 

I'll :shipit: now.

**Comment by BillWagner** [comment]
@pkulikov Thanks for the detailed notes with respect to this work and the proposed consolidation project.

I'll finish my notes and creating issues for the consolidation project today. Before anyone starts work on them, I'd like to get feedback on the entire proposal from the language team and others in the community. After that's approved, I welcome any help you want to provide. In the meantime, making the existing content better helps, and will make those tasks easier.

Thanks again for all your help.

**Pull request merged**